src/engine/Settings.cs

```diff
@@ -122,6 +124,12 @@ public enum AntiAliasingMode
     [JsonProperty]
     public SettingValue<float> RenderScale { get; private set; } = new(1.0f);
 
+    /// <summary>
+    /// Selected window index for the game window.
+    /// </summary>
+    [JsonProperty]
+    public SettingValue<int> SelectedDisplayIndex { get; set; } = new(DisplayServer.WindowGetCurrentScreen());
```

Comment:
I really think the default should be -1 and it should mean "auto" as on non-Windows platform it's going to be annoying if the game forces itself on a specific screen.